File: task_1.java

public class Main
{
    public static void main(String[] args) {
        separateAndPrintLetters("HeLLoWoRLD");
    }
    public static void separateAndPrintLetters(String input) {

        StringBuilder upperCase = new StringBuilder();
        StringBuilder lowerCase = new StringBuilder();

        for (char c : input.toCharArray()) {
            if (Character.isUpperCase(c)) {
                upperCase.append(c);
            } else if (Character.isLowerCase(c)) {
                lowerCase.append(c);
            }
        }

        System.out.println("Заглавные: " + upperCase.toString());
        System.out.println("Строчные: " + lowerCase.toString());
    }
}

File: task_1.cpp
#include <bits/stdc++.h>
#include <iostream>
#include <string>
#include <cctype>

using namespace std;

void separateAndPrintLetters(const string& input) {
    string upperCase;
    string lowerCase;

    for (char c : input) {
        if (isupper(c)) {
            upperCase += c;
        } else if (islower(c)) {
            lowerCase += c;
        }
    }

    cout << "Заглавные: " << upperCase << endl;
    cout << "Строчные: " << lowerCase << endl;
}

int main() {
    // Вызываем функцию с примером строки
    separateAndPrintLetters("HeLLoWoRLD");
    
    return 0;
}
